src/radical/pilot/raptor/master.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, cfg=None):
         self._term       = mt.Event()  # termination signal
         self._thread     = None        # run loop
 
-        self._hb_freq    = 10          # check worker heartbetas every n seconds
-        self._hb_timeout = 15          # consider worker dead after 15 seconds
+        self._hb_freq    = 1000        # check worker heartbetas every n seconds
```

Comment:
maybe set every 100s?

@andre-merzky  regarding to it, should we also move `time.sleep(self._hb_freq)` after `for uid in lost`-loop?

https://github.com/radical-cybertools/radical.pilot/blob/746d54b6c05caac193dc716e1688b97475381a07/src/radical/pilot/raptor/master.py#L555-L561

Reply:
Yep, that makes sense.

Reply:
@AymenFJA just to make clear, you increased the frequency of checks, thus you check the status every 1000s, but actual timeout value is 150s. I kinda thought to have at least one check during timeout time range, maybe then to have  `hb_freq = 500` and `hb_timeout = 1000`. @andre-merzky is this correct?

Reply:
Indeed - the check frequency should be usually lower than the heartbeat frequency.  The timeout value is the important one to increase in this context.